js/ui/html_editor/utils/table_helper.js

```diff
@@ -29,8 +29,13 @@ function hasEmbedContent(module, selection) {
     return !!selection && module.quill.getText(selection).trim().length < selection.length;
 }
 
-function unfixTableWidth($table) {
-    $table.css('width', 'initial');
+function unfixTableWidth($table, tableData) {
+    const unfixValue = 'initial';
+    if(tableData && tableData[0]) {
+        tableData[0].format('tableWidth', unfixValue);
```

Comment:
```suggestion
function unfixTableWidth($table, tableData = []) {
    const unfixValue = 'initial';
    if(tableData.length) {
        tableData[0].format('tableWidth', unfixValue);
```

And it's really necessary to pass all data? 

```suggestion
function unfixTableWidth($table, tableBlot) {
    const unfixValue = 'initial';
    if(tableBlot) {
        tableBlot.format('tableWidth', unfixValue);
```